### ISSUE TITLE
Xcode 14.3 beta not compiling

### ISSUE THREAD

**Issue by cohen72** [michaeltyson/TPCircularBuffer#38]
Any ideas how to resolve this issue?

`ld: file not found: /Applications/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/arc/libarclite_iphoneos.a
clang: error: linker command failed with exit code 1 (use -v to see invocation)`

Podfile:

```
source 'https://github.com/CocoaPods/Specs.git'
platform :ios, '15.0'
use_frameworks!

target '<AppName>' do
  pod 'TPCircularBuffer', '1.6.1'
  pod 'Mixpanel', :git => 'https://github.com/nanit/mixpanel-iphone'
  pod 'Intercom'
end

```

**Comment by mman** [comment]
Use SPM Luke!

**Comment by cohen72** [comment]
@mman  
> Use SPM Luke!

Yes I would, however since we're using KMM (Kotlin Multiplatform Mobile), it's accessing the lib via Cocoapods ... I'll have to look into how to use it as a swift package instead, together with KMM

**Comment by tetujin** [comment]
Hi @cohen72, I have faced the same issue on Xcode 14.3 with Cocoapods. 

You can fix this by changing "Minimum Deployments" to more than 11.0 from 8.0. 
The parameter is located on Xcode -> TAEGETS -> TPCircularBuffer -> General -> Minumum Deployments
